.github/workflows/test.yaml

```diff
@@ -366,6 +366,56 @@ jobs:
       name: Deploy to S3
       run: ./build-support/bin/deploy_to_s3.py
     timeout-minutes: 65
+  build_wheels_aarch64:
```

Comment:
We generate our CI files. You'll want to update `build-support/bin/generate_github_workflows.py`, then run `./pants run build-support/bin/generate_github_workflows.py`.